packages/ui-progress/src/ProgressCircle/index.tsx

```diff
@@ -135,20 +132,17 @@ class ProgressCircle extends Component<
 
     const valueText =
       typeof formatScreenReaderValue === 'function' &&
-      formatScreenReaderValue({ valueNow, valueMax })
+      formatScreenReaderValue({ valueNow: valueNow!, valueMax: valueMax! })
     // consolidating the label and aria-valuetext to put in aria-label because
     // NVDA does not read aria-valuetext: https://github.com/nvaccess/nvda/issues/913
     // But leaving aria-valuetext because JAWS ignores aria-label
     const labelAndValueText = `${screenReaderLabel} ${valueText}`
 
     const value = callRenderProp(renderValue, { valueNow, valueMax })
 
-    // @ts-expect-error ts-migrate(2339) FIXME: Property 'animateOnMount' does not exist on type '... Remove this comment to see the full error message
-    const style = this.state.animateOnMount
-      ? null
-      : {
```

Comment:
This is possibly another bug :D But I checked in v7, same issue here, and there was no complaint about it, so hopefully not an issue.

Reply:
It is indeed an unused state variable. Probably no one needs it

Reply:
I meant that maybe ages ago this was needed for the "animate on mount" style, but later the state variable got renamed and this wasn't refactored here by accident. But even when using the current state variable I couldn't see any difference, so probably it is not a problem to remove it (instead of renaming the variable).